hydroshare/wsgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
```

Comment:
Reduce number of places env set for consolidation and ease of management/troubleshooting